rhui_provision/install_monitor_service.yml

```diff
@@ -0,0 +1,10 @@
+- name: Install Packages for Monitoring
+  hosts: MON_SERV 
```

Comment:
I tend to prefer fully written out variable names in Ansible playbooks when it makes sense. In this case I'm wondering whether MON_SERV means "Monitoring Service" or "Monitoring Server".

It's a nitpick, but I think it helps code readability in the long run.